components/rabbitmq/scripts/create.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+
+import importlib
+import os
+from subprocess import check_output
+
+utils_path = check_output([
+    'ctx', 'download-resource', 'components/utils.py',
+    os.path.join(os.path.dirname(__file__), 'utils.py')])
+ctx = utils = importlib.import_module('utils')
+
+
+config_path = "components/rabbitmq/config"
+
+rabbitmq_endpoint_ip = ctx.node.properties('rabbitmq_endpoint_ip')
+
+
+def install_rabbitmq():
+    # Sources
+    erlang_source_url = ctx.node.properties('erlang_rpm_source_url')
```

Comment:
while I agree that we should definitely come up with something cleverer than having all vars as globals, I think we should at least be consistent in the beginning.

please move all service definitions (paths, ports, ctx properties, etc..) to be global at the top and we'll come up with a good solution later on.

See influx as reference. thanks